tmuxp/cli.py

```diff
@@ -1186,10 +1186,13 @@ def command_convert(confirmed, config):
     """Convert a tmuxp config between JSON and YAML."""
 
     _, ext = os.path.splitext(config)
-    if 'json' in ext:
+    ext = ext.lower()
+    if ext == '.json':
         to_filetype = 'yaml'
-    elif 'yaml' in ext:
+    elif ext in ['.yaml', '.yml']:
         to_filetype = 'json'
+    else:
+        raise click.BadParameter('Unknown filetype: %s (valid: [.json, .yaml, .yml])' % (ext, ))
```

Comment:
It may be helpful to run `black` here

in re: `./tmuxp/cli.py:1195:89: E501 line too long (96 > 88 characters)`

We've recently added a precommit bot so this should get easier

Reply:
ok